### PR TITLE
YONK-721: Complete Status added in Passed Users API

### PR DESCRIPTION
**Pull request by aamishbaloch**
**complete_status** added in Passed Users API.

### CODE REVIEW

**Comment by aamishbaloch** [comment]
@ziafazal Can you please review this?

File: edx_solutions_api_integration/courses/tests.py [hunk 1]
```diff
@@ -1275,6 +1271,37 @@ def test_courses_users_list_courses_passed(self):
...
+
+        module = self.get_module_for_user(users[0], course, course.homework_assignment)
+        grade_dict = {'value': 1, 'max_value': 1, 'user_id': users[0].id}
+        module.system.publish(module, 'grade', grade_dict)
+        module = self.get_module_for_user(users[1], course, course.homework_assignment)
+        grade_dict = {'value': 1, 'max_value': 1, 'user_id': users[1].id}
+        module.system.publish(module, 'grade', grade_dict)
+
+        user_grade, user_proforma_grade = 0.9, 0.91
+        StudentGradebook.objects.update_or_create(
```
Comment (ziafazal) [inline edx_solutions_api_integration/courses/tests.py]: I don't think we need to create gradebook here `module.system.publish(module, 'grade', grade_dict)` should do it. I mean we should eliminate either of two statements in this test.

File: edx_solutions_api_integration/courses/serializers.py [hunk 2]
```diff
@@ -89,3 +90,22 @@ class OrganizationCourseSerializer(CourseSerializer):
     class Meta(object):
         """ Serializer/field specification """
         fields = ('id', 'name', 'number', 'org', 'start', 'end', 'due', 'enrolled_users', )
+
+
+class PassedUserSerializer(serializers.Serializer):
```
Comment (ziafazal) [inline edx_solutions_api_integration/courses/serializers.py]: Can we name it UserGradebookSerializer?

File: edx_solutions_api_integration/courses/serializers.py [hunk 3]
```diff
@@ -89,3 +90,22 @@ class OrganizationCourseSerializer(CourseSerializer):
...
+    id = serializers.IntegerField(source='user__id')
+    email = serializers.CharField(source='user__email')
+    username = serializers.CharField(source='user__username')
+    first_name = serializers.CharField(source='user__first_name')
+    last_name = serializers.CharField(source='user__last_name')
+    is_active = serializers.BooleanField(source='user__is_active')
+    created = serializers.DateTimeField(source='user__date_joined')
+    complete_status = serializers.SerializerMethodField()
+
+    def get_complete_status(self, record):
```
Comment (ziafazal) [inline edx_solutions_api_integration/courses/serializers.py]: rename `record` to `gradebook`?

File: edx_solutions_api_integration/courses/views.py [hunk 4]
```diff
@@ -1164,8 +1165,18 @@ def get_queryset(self):
         course_key = get_course_key(course_id)
         exclude_users = get_aggregate_exclusion_user_ids(course_key)
 
-        queryset = StudentGradebook.get_passed_users(
+        queryset = StudentGradebook.get_passed_users_gradebook(
             course_key, exclude_users=exclude_users, org_ids=org_ids, group_ids=group_ids
+        ).values(
```
Comment (ziafazal) [inline edx_solutions_api_integration/courses/views.py]: why we need to convert this queryset to `values`?

**Pull request merged**